.travis.yml

```diff
@@ -24,16 +24,17 @@ jobs:
   include:
     - name: "Rust: stable - Examples"
       rust: stable
-      services:
-        - docker
-      install:
-        - docker pull trufflesuite/ganache-cli:v6.7.0
       script:
-        - docker run -d -p 9545:8545 trufflesuite/ganache-cli:v6.7.0
+        - |
+          mkfifo /tmp/truffle-in
+          tail -f /tmp/truffle-in | yarn --cwd examples/truffle start > /dev/null &
         - cargo run --example async
         - cargo run --example linked
         - cargo run --package examples-generate
         - cargo run --example rinkeby
+        - |
+          kill %1
```

Comment:
You are such a good citizen. What would happen if you didn't kill here?

Reply:
No idea! I assume the shell would exit and kill all background jobs.